Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPostCampaignViewControllerTests.swift

```diff
@@ -259,38 +158,41 @@ final class NoShippingPostCampaignViewControllerTests: TestCase {
       }
   }
 
-  func testView_HasAddOns() {
+  func testView_AddOns_LoggedOut() {
```

Comment:
The previous version of this test didn't actually show add-ons, so I've deleted the corresponding snapshots and made an entirely new test instead.